veth-create-host.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+SERVER=$1
+CLIENT=$2
+
+if [ -z $SERVER ]; then
+    echo "Server NetNS must be specified."
```

Comment:
This can be confusing. I think it should be `Server Docker container name`, right?

Reply:
will fix